adobe-apiplatform/user-sync.py
# Update connector_umapi.py

##Summary

##Testing Steps

use additional_groups with groups auto-create: True
get a 504 on first call that gets all groups from admin console
continue with the sync and see the sending sync start signal before initial GET groups page 0 call

Fixes #813
